relay-redis/src/config.rs

```diff
@@ -33,6 +55,10 @@ impl Default for RedisConfigOptions {
             idle_timeout: 60,
             read_timeout: 3,
             write_timeout: 3,
```

Comment:
These are unused now, I think?

Reply:
I will follow up with a cleanup, I have a ticket for it.